### PR TITLE
completed

### PR DESCRIPTION
**Pull request by Touseef-ahmad-arbisoft**

### CODE REVIEW

File: myForm.html [hunk 1]
```diff
@@ -5,10 +5,47 @@
...
     <title>My Form</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 
 <body>
-
+    <form action="" id="survey-form">
+        <h1 id="title">My Survey Form</h1>
+        <p id="description">Thank you for your precious time.</p>
+        <fieldset id="fs1">
```
Comment (Touseef-ahmad-arbisoft) [inline myForm.html]: dont use vague ids, use something more meaningful, like survey-form__fieldset__name, you can reference something like  -> https://css-tricks.com/bem-101/

File: myForm.html [hunk 2]
```diff
@@ -5,10 +5,47 @@
...
+                    <option value="2">Business Owner</option>
+                    <option value="3">Employed</option>
+                    <option value="4">UnEmployed</option>
+                </select>
+            </label>
+        </fieldset>
+        <fieldset>
+            <legend>Gender</legend>
+            <label for=""><input class="inline" type="radio" name="gen" id="" value="male"> Male</label>
+            <label for=""><input class="inline" type="radio" name="gen" id="" value="female"> Female</label>
```
Comment (Touseef-ahmad-arbisoft) [inline myForm.html]: missing for=""

File: myForm.html [hunk 3]
```diff
@@ -5,10 +5,47 @@
...
+        <p id="description">Thank you for your precious time.</p>
+        <fieldset id="fs1">
+            <label id="name-label" for="name">Name <input type="text" id="name" placeholder="Enter your Name"
+                    required></label>
+            <label id="email-label" for="email">Email <input type="email" id="email" placeholder="Enter your Email"
+                    required></label>
+            <label id="number-label" for="number">Age <input type="number" name="" id="number" min="13" max="120"
+                    placeholder="Age" required></label>
+            <label for="">Which role describe you well
+                <select id="dropdown">
```
Comment (Touseef-ahmad-arbisoft) [inline myForm.html]: indentation

File: myForm.html [hunk 4]
```diff
@@ -5,10 +5,47 @@
...
+            <label for="travel"><input type="checkbox" id="travel" value="travel"> Travelling</label>
+        </fieldset>
+        <fieldset>
+            <label for="">Additional Comments<textarea name="" id="" cols="30" rows="3"></textarea></label>
+        </fieldset>
+        <input type="submit" id="submit" value="Submit">
+    </form>
 </body>
 
 </html>
```
Comment (Touseef-ahmad-arbisoft) [inline myForm.html:55]: blank line at the end of file.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    font-family: tahoma;
+    min-width: 350px;
+    width: 100%;
+    max-width: 500px;
+    height: 100vh;
+    margin: auto;
+}
+
+#survey-form {
+    padding: 20px 20px
```
Comment (Touseef-ahmad-arbisoft) [inline styles.css]: no ; colon added

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+h1,
+p {
+    margin: 1em auto;
```
Comment (Touseef-ahmad-arbisoft) [inline styles.css:18]: why using em?

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+#dropdown,
+#fs1 input {
+    display: block;
+    width: 100%;
+    padding: 2px 3px;
+    font-size: 16px;
+    margin: 5px 0;
+}
+
+input[type="submit"] {
```
Comment (Touseef-ahmad-arbisoft) [inline styles.css]: try to be more specific, add some classname or id on submit button.

File: myForm.html [hunk 8]
```diff
@@ -5,10 +5,47 @@
...
+            <legend>Interests</legend>
+            <label for="animals"><input type="checkbox" id="animals" value="animals"> Animals</label>
+            <label for="music"><input type="checkbox" id="music" value="music"> Music</label>
+            <label for="tech"><input type="checkbox" id="tech" value="tech"> Technology</label>
+            <label for="travel"><input type="checkbox" id="travel" value="travel"> Travelling</label>
+        </fieldset>
+        <fieldset>
+            <label for="">Additional Comments<textarea name="" id="" cols="30" rows="3"></textarea></label>
+        </fieldset>
+        <input type="submit" id="submit" value="Submit">
```
Comment (Touseef-ahmad-arbisoft) [inline myForm.html]: use survey-form__submit

File: styles.css (hunk 6 above)
Comment (Zubair-Rajpoot) [reply styles.css:18]: we use 'em' if we want to make our design flexible and scaleable according to user preferences.
we should use 'em' with margin, padding and font-sizes for better and responsive design.
we can use 'px' for fixed sizes i.e: borders and icons etc.

**Pull request merged**